packages/verkle/src/verkleTree.ts

```diff
@@ -290,11 +305,9 @@ export class VerkleTree {
     await this.saveStack(putStack)
   }
 
-  async del(key: Uint8Array): Promise<void> {
-    const stem = key.slice(0, 31)
-    const suffix = key[key.length - 1]
-    this.DEBUG && this.debug(`Stem: ${bytesToHex(stem)}; Suffix: ${suffix}`, ['DEL'])
-    await this.put(key, createDeletedLeafValue())
+  async del(stem: Uint8Array, suffixes: number[]): Promise<void> {
+    this.DEBUG && this.debug(`Stem: ${bytesToHex(stem)}; Suffix(es): ${suffixes}`, ['DEL'])
+    await this.put(stem, suffixes, new Array(suffixes.length).fill(createDeletedLeafValue()))
```

Comment:
I think it would be more efficient to first declare `deletedLeafValue = createDeletedLeafValue()`, and then fill with that declared value `fill(deletedLeafValue)` rather than create each time. Unless we have a reason to want different objects instantiations?

Reply:
Well, if you declare it once and then reference it across all the deleted suffixes, I'm pretty sure you just get that references to one Uint8Array of 32 0s.  I'm assuming that is problematic.